contracts/src/v0.8/dev/ConfirmedOwnerWithProposal.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/OwnableInterface.sol";
+
+/**
+ * @title The ConfirmedOwner contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract ConfirmedOwnerWithProposal is OwnableInterface {
```

Comment:
Just a rename to be more descriptive?

Reply:
The difference between `ConfirmedOwner` and `ConfirmedOwnerWithProposal` is that the deployer can set a proposed owner during construction. 

So `ConfirmedOwner` still exists and extends `ConfirmedOwnerWithProposal` without setting a proposal on construction.